diff --git a/eslint/rules/react.js b/eslint/rules/react.js
index 71fc5f5..b29c017 100644
--- a/eslint/rules/react.js
+++ b/eslint/rules/react.js
@@ -13,7 +13,7 @@ const config = {
     "react/jsx-no-target-blank": ["error", { allowReferrer: true }],
     "react/jsx-no-useless-fragment": ["warn", { allowExpressions: true }],
     "react/jsx-pascal-case": ["warn"],
-    "react/jsx-sort-props": ["warn", { reservedFirst: ["key"] }],
+    "react/jsx-sort-props": ["off"],
     "react/no-array-index-key": ["warn"],
     "react/no-unknown-property": ["off"],
     "react/no-unstable-nested-components": ["error"],
